As a user, I to make corrections to the code and/or wiki to better reflect reality when employing combination, operation, and aggregation

The full test declaration is below; it was picked at random and then I started toying with different `combination` options. 

Specifically, `aggregation` "none" is not deserialized successfully even though the schema indicates its valid. For example, 

```yaml
    combination: 
#      operation: intersection
      aggregation: none
```

yields this exception:

> Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type wres.config.yaml.components.TimeWindowAggregation from String "none": not one of the values accepted for Enum class: [maximum, average, minimum]

Also, when `aggregation` is valid, the validation error implies that a default `operation` of `union` is selected:

```yaml
    combination:
#      operation: intersection
      aggregation: maximum
```

yields this validation error (and only this error):

> Event detection was declared with an 'operation' of 'union' and an aggregation method of 'maximum', which is not valid. Please remove the 'aggregation' method, declare an 'aggregation' method of 'none' or change the 'operation' to 'intersection'. An explicit 'aggregation' method is only valid when the 'operation' is 'intersection'.

That message appears to indicate that a default of "union" for operation is assumed. I don't think that's true based on the wiki, and based on validation errors for the declaration in [this comment](https://github.com/NOAA-OWP/wres/issues/382#issuecomment-2665964647).

I think this ticket can be resolved when:

1. "none" is properly parsed for the `aggregation`.
2. If `operation` is not included, a validation error message tells me it needs to be included **in all cases**, matching the wiki.  This is not currently true if I specify `aggregation: maximum` as described above. 

Thanks,

Hank

==========

```yaml
label: Testing Event Based
observed:
  label: OBS Streamflow
  sources: /home/ISED/wres/wresTestData/issue92087/inputs/ABRN1_QME.xml
  variable: QME
  feature_authority: nws lid
  type: observations
  time_scale:
    function: mean
    period: 24
    unit: hours

predicted:
  label: "19161749 RetroSim CSVs"
  sources:
  - /home/ISED/wres/nwm_3_0_retro_simulations/wfo/OAX/19161749_nwm_3_0_retro_wres.csv.gz
  variable: streamflow
  feature_authority: nwm feature id
  type: simulations

features:
  - {observed: ABRN1, predicted: '19161749'}

time_scale:
  function: mean
  period: 24
  unit: hours

#reference_dates:
#  minimum: 2000-08-07T23:00:00Z
#  maximum: 2015-08-08T23:00:00Z

#event_detection: observed
event_detection:
  dataset:
    - observed
    - predicted
  parameters:
    window_size: 4800
    start_radius: 0
    half_life: 480
    minimum_event_duration: 0
    duration_unit: hours
    combination:
      operation: union
      aggregation: 'none'

metrics:
  - time to peak error
  - sample size
  - pearson correlation coefficient

output_formats:
  - csv2
  - png
```

--- a/src/wres/pipeline/pooling/EventsGenerator.java
+++ b/src/wres/pipeline/pooling/EventsGenerator.java
@@ -259,15 +259,15 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
             }
         }
 
-        LOGGER.info( "Detected {} events across all datasets for feature group {} when forming the {}.",
-                     events.size(),
-                     featureGroup.getName(),
-                     combination );
-
         // Aggregate any intersecting events, as needed, but only if combination/intersection happened
         Set<TimeWindowOuter> aggregated = events;
         if ( detectionAttemptedCount > 1 ) // Combination/intersection happened
         {
+            LOGGER.info( "Detected {} events across all datasets for feature group {} when forming the {}.",
+                         events.size(),
+                         featureGroup.getName(),
+                         combination );
+
             aggregated = this.aggregateEvents( events,
                                                detection.parameters()
                                                         .aggregation(),
@@ -306,32 +306,14 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
         RetrieverFactory<Double, Double, Double> eventRetriever = details.eventRetriever();
 
         // Get any valid time constraints on retrieval, accounting for the timescale
-        TimeWindowOuter timeWindow = TimeWindowOuter.of( MessageUtilities.getTimeWindow() );
-        EvaluationDeclaration declaration = details.declaration();
-        TimeInterval validDates = declaration.validDates();
-        if ( Objects.nonNull( validDates ) )
-        {
-            Instant minimumInstant = validDates.minimum();
-            Instant maximumInstant = validDates.maximum();
-            Timestamp minimum = MessageUtilities.getTimestamp( minimumInstant );
-            Timestamp maximum = MessageUtilities.getTimestamp( maximumInstant );
-            TimeWindow adjusted = timeWindow.getTimeWindow()
-                                            .toBuilder()
-                                            .setEarliestValidTime( minimum )
-                                            .setLatestValidTime( maximum )
-                                            .build();
-            TimeWindowOuter adjustedOuter = TimeWindowOuter.of( adjusted );
-
-            // Adjust for timescale
-            timeWindow = TimeWindowSlicer.adjustTimeWindowForTimeScale( adjustedOuter, details.desiredTimeScale() );
-        }
+        TimeWindowOuter timeWindow = this.getTimeWindow( details.declaration(), details.desiredTimeScale() );
 
         // Get the features for the dataset orientation, which is one of the main datasets (observed, predicted or
         // baseline), not a covariate. The covariate features must be further derived by correlating the feature names
         // whose features have the same feature authority as the covariate dataset, which is a requirement. See below.
         Set<Feature> features = this.getFeatures( featureGroup.getFeatures(), featureGetter );
 
-        LOGGER.info( "Getting time-series data to perform event detection for the following features: {}", features );
+        LOGGER.debug( "Getting time-series data to perform event detection for the following features: {}", features );
 
         switch ( details.dataset() )
         {
@@ -340,11 +322,12 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
                 Stream<TimeSeries<Double>> series = eventRetriever.getLeftRetriever( features, timeWindow )
                                                                   .get();
 
-                Set<TimeWindowOuter> innerEvents = series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
-                                                                                                              details,
-                                                                                                              this.leftUpscaler() )
-                                                                            .stream() )
-                                                         .collect( Collectors.toSet() );
+                Set<TimeWindowOuter> innerEvents =
+                        series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
+                                                                                   details,
+                                                                                   this.leftUpscaler() )
+                                                 .stream() )
+                              .collect( Collectors.toSet() );
                 LOGGER.info( DETECTED_EVENTS_IN_THE_DATASET,
                              innerEvents.size(),
                              EventDetectionDataset.OBSERVED,
@@ -356,11 +339,12 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
                 Stream<TimeSeries<Double>> series = eventRetriever.getRightRetriever( features, timeWindow )
                                                                   .get();
 
-                Set<TimeWindowOuter> innerEvents = series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
-                                                                                                              details,
-                                                                                                              this.rightUpscaler() )
-                                                                            .stream() )
-                                                         .collect( Collectors.toSet() );
+                Set<TimeWindowOuter> innerEvents =
+                        series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
+                                                                                   details,
+                                                                                   this.rightUpscaler() )
+                                                 .stream() )
+                              .collect( Collectors.toSet() );
                 LOGGER.info( DETECTED_EVENTS_IN_THE_DATASET,
                              innerEvents.size(),
                              EventDetectionDataset.PREDICTED,
@@ -372,11 +356,12 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
                 Stream<TimeSeries<Double>> series = eventRetriever.getBaselineRetriever( features, timeWindow )
                                                                   .get();
 
-                Set<TimeWindowOuter> innerEvents = series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
-                                                                                                              details,
-                                                                                                              this.baselineUpscaler() )
-                                                                            .stream() )
-                                                         .collect( Collectors.toSet() );
+                Set<TimeWindowOuter> innerEvents =
+                        series.flatMap( s -> this.adjustTimeSeriesAndDetectEvents( s,
+                                                                                   details,
+                                                                                   this.baselineUpscaler() )
+                                                 .stream() )
+                              .collect( Collectors.toSet() );
                 LOGGER.info( DETECTED_EVENTS_IN_THE_DATASET,
                              innerEvents.size(),
                              EventDetectionDataset.BASELINE,
@@ -428,6 +413,39 @@ record EventsGenerator( TimeSeriesUpscaler<Double> leftUpscaler,
         }
 
         return Collections.unmodifiableSet( events );
+    }
+
+    /**
+     * Returns the time window for retrieval of the time-series data used in event detection, accounting for any valid
+     * time constraints and desired time-scale.
+     *
+     * @param declaration the declaration
+     * @param desiredTimeScale the desired time scale
+     * @return the time window
+     */
+
+    private TimeWindowOuter getTimeWindow( EvaluationDeclaration declaration, TimeScaleOuter desiredTimeScale )
+    {
+        TimeWindowOuter timeWindow = TimeWindowOuter.of( MessageUtilities.getTimeWindow() );
+        TimeInterval validDates = declaration.validDates();
+        if ( Objects.nonNull( validDates ) )
+        {
+            Instant minimumInstant = validDates.minimum();
+            Instant maximumInstant = validDates.maximum();
+            Timestamp minimum = MessageUtilities.getTimestamp( minimumInstant );
+            Timestamp maximum = MessageUtilities.getTimestamp( maximumInstant );
+            TimeWindow adjusted = timeWindow.getTimeWindow()
+                                            .toBuilder()
+                                            .setEarliestValidTime( minimum )
+                                            .setLatestValidTime( maximum )
+                                            .build();
+            TimeWindowOuter adjustedOuter = TimeWindowOuter.of( adjusted );
+
+            // Adjust for timescale
+            timeWindow = TimeWindowSlicer.adjustTimeWindowForTimeScale( adjustedOuter, desiredTimeScale );
+        }
+
+        return timeWindow;
     }
 
     /**

--- a/wres-config/nonsrc/schema.yml
+++ b/wres-config/nonsrc/schema.yml
@@ -639,6 +639,8 @@ definitions:
         "$ref": "#/definitions/EventDetectionCombinationEnum"
       aggregation:
         "$ref": "#/definitions/EventDetectionAggregationEnum"
+    required:
+      - operation
 
   CrossPair:
     title: Cross-pairing of time-series for consistency
@@ -1873,15 +1875,14 @@ definitions:
   EventDetectionAggregationEnum:
     title: The method for aggregating events across data sources.
     description: "Identifies the method for aggregating events across different 
-      time-series data sources, such as observed and predicted. The strategy 
-      'none' applies no aggregation. The 'maximum' forms the maximum period 
-      across each set of intersecting events. The 'minimum' forms the minimum 
-      period across each set of intersecting events or the temporal 
-      intersection. The 'average' forms the average of the start datetimes and 
-      the end datetimes for each set of intersecting events."
+      time-series data sources, such as observed and predicted. The 'maximum' 
+      forms the maximum period across each set of intersecting events. The 
+      'minimum' forms the minimum period across each set of intersecting events 
+      or the temporal intersection. The 'average' forms the average of the 
+      start datetimes and the end datetimes for each set of intersecting 
+      events."
     type: string
     enum:
-      - none
       - maximum
       - minimum
       - average

--- a/wres-config/src/wres/config/yaml/DeclarationValidator.java
+++ b/wres-config/src/wres/config/yaml/DeclarationValidator.java
@@ -2524,8 +2524,7 @@ public class DeclarationValidator
                                                                              .toString()
                                                                              .toLowerCase()
                                                                  + "', which is not valid. Please remove the "
-                                                                 + "'aggregation' method, declare an 'aggregation' "
-                                                                 + "method of 'none' or change the 'operation' to "
+                                                                 + "'aggregation' method or change the 'operation' to "
                                                                  + "'intersection'. An explicit 'aggregation' method "
                                                                  + "is only valid when the 'operation' is "
                                                                  + "'intersection'." )

